src/components/notifications/Onesender.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div class="mb-3">
+        <label for="host-onesender" class="form-label">{{ $t("HostOnesender") }}</label>
+        <input
+            id="host-onesender"
+            v-model="$parent.notification.onesenderURL"
+            type="url"
+            placeholder="https://xxxxxxxxxxx.com/api/v1/messages"
+            pattern="https?://.+"
+            class="form-control"
+            required
+        />
+    </div>
+
+    <div class="mb-3">
+        <label for="receiver-onesender" class="form-label">{{ $t("TokenOnesender") }}</label>
+        <input
```

Comment:
Please use `HiddenInput` for secret variables instead ^^

Reply:
i use password type input